tools/ColumnTable.lua

```diff
@@ -1,5 +1,5 @@
 --------------------------------------------------------------------------
--- This class is takes a array of strings and produced a multi-column output.
+-- This class takes a array of strings and produced a multi-column output.
```

Comment:
The expression `a array` appears in some files, and actually this file has a variable `a`, but perhaps they should be `an array`.